packages/strapi-provider-email-nodemailer/package.json

```diff
@@ -0,0 +1,52 @@
+{
+  "name": "strapi-provider-email-nodemailer",
+  "version": "3.0.0",
+  "description": "Nodemailer provider for Strapi 3",
+  "main": "./lib",
+  "directories": {
+    "lib": "./lib"
+  },
+  "dependencies": {
+    "lodash": "4.17.19",
+    "nodemailer": "6.3.0"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/strapi/strapi.git"
```

Comment:
Shouldn't it be either `git://` or `https://`.

Reply:
Yes, thanks, according to https://docs.npmjs.com/cli/v6/configuring-npm/package-json#repository this is just an ordinary URL for humans.

Reply:
What about the missing keys? I removed key `strapi` because of this comment: https://github.com/strapi/strapi/pull/8459#discussion_r512280074

Reply:
All the provider do have the `isProvider` except for `strapi-provider-upload-rackspace` & `strapi-provider-email-amazon-ses`, but I guess that this is something old and not used anymore. @derrickmehaffy?

Reply:
I've never noticed it before tbh @MattieBelt but worst case we can make our own PRs to fix any issues before release. Don't want to stall this PR just for minor stuff ;)